test/lib/concerns/base_template_scope_extensions_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'katello_test_helper'
+require 'foreman/renderer/scope/provisioning'
+
+module Katello
+  class BaseTemplateScopeExtensionsTest < ActiveSupport::TestCase
+    def setup
+      @errata = katello_errata(:security)
+    end
+
+    def test_errata
+      source = Foreman::Renderer::Source::String.new(
```

Comment:
Are tests failing due to a missing include for this?

Reply:
I think so, lets try this :)